workflow/cron/operator.go

```diff
@@ -191,6 +191,10 @@ func (woc *cronWfOperationCtx) terminateOutstandingWorkflows(ctx context.Context
 		woc.log.Infof("stopping '%s'", wfObjectRef.Name)
 		err := util.TerminateWorkflow(ctx, woc.wfClient, wfObjectRef.Name)
 		if err != nil {
+			if errors.IsNotFound(err) {
```

Comment:
Not able to reproduce locally.   code will log the warning if the terminate workflow is already deleted